cli/cmd/admin/start.go

```diff
@@ -229,7 +229,7 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 			// Run tasks
 			err = group.Wait()
 			if err != nil {
-				logger.Error("crashed", zap.Error(err))
+				logger.Fatal("crashed", zap.Error(err))
```

Comment:
OOPS. I wanted process to exit with error so that I could get an error and close other processes in case of port mismatches. I can find some other way as well.